bindings/python/pyiree/rt/system_api.py

```diff
@@ -125,6 +133,41 @@ def _get_global_config():
   return _global_config
 
 
+def _normalize_numpy(array: np.ndarray,
```

Comment:
Can you provide a comment to this method with the the transformations that are being done? I'm not quite sure I'm following the 'single_precision' part.